HopsanGenerator/src/generators/HopsanModelicaGenerator.cpp

```diff
@@ -200,15 +200,15 @@ void HopsanModelicaGenerator::parseModelicaModel(QString code, QString &typeName
             }
             else if(words.at(0).startsWith("annotation("))        //"annotation" keyword
             {
-                QString tempLine = lines[l];
+                QString tempLine = line;
                 tempLine.remove(" ");
                 int idx = tempLine.indexOf("hopsanCqsType=");
                 cqsType = tempLine.at(idx+15);
             }
             else if(words.at(0) == "parameter")         //"parameter" keyword
             {
                 QString name = words.at(2).section("(",0,0);
-                QString unit = lines.at(l).section("unit=",1,1).section("\"",1,1);
+                QString unit = line.section("unit=",1,1).section("\"",1,1);
```

Comment:
I do not know modelica, but what about requirements on = does it have to be attached directly to the name (unit) or would it be valid modelica code with a space, like in 'unit ='

Reply:
Sorry, this was not supposed to be in this PR. I accidentally included a commit from another branch. But to answer the question, I don't know. All examples I have seen have the equality symbol attached to the name, but I guess spaces are allowed too. I will fix this in the other branch.

Reply:
OK